src/_pytest/terminal.py

```diff
@@ -84,6 +84,20 @@ def pytest_addoption(parser):
         dest="verbose",
         help="increase verbosity.",
     )
+    group._addoption(
```

Comment:
would using a store_true/store_false be more appropriate here? assuming we expect this only once on the command line (unlike -v) we behaves differently based on the count

Reply:
done